shacl12-overview/index.html

```diff
@@ -384,13 +472,6 @@ <h2>Security and Privacy Considerations</h2>
 			</p>
 		</section>
 
-		<section id="validator" class="appendix normative">
-			<h2>Validator</h2>
-			<p>
-				The shapes graph of SHACL-SHACL is available as an RDF resource at <a href="http://www.w3.org/ns/shacl-shacl">http://www.w3.org/ns/shacl-shacl</a>.
```

Comment:
What will this link be?  I missed some of the nuance about whether we need to have "shacl12" somewhere in this path.

If we need to reflect SHACL 1.2 outside of the link and instead inlined in the graph-file, would `owl:versionIRI` be an option?

Please feel free to split this comment out to its own Issue if that'll be a big discussion.  I somewhat suspect it will be.

Reply:
This info's location is actually already superseded with the Appendix called "SHACL-SHACL", not "Validator", but the question about the IRI remains.

There is a plan to harmonise all SHACL namespaces into one, after development slows down/ceases, but I'm not sure if SHACL-SHACL will live in there with the rest of SHACL's definitions. I think we will retain `/shacl-shacl`, since it's already there.

I suggest we will make SHACL-SHACL all (the comprehensive 1.2 validator) be the default at `http://www.w3.org/ns/shacl-shacl`, that we create other IRIs for the part validators and that we point a `versionIRI`, as you say, at the soon-to-be old validator from 1.0.